<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSurface</class>
 <widget class="QDialog" name="CreateSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Surface</string>
  </property>
  <widget class="QLabel" name="MaxLenghL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Triangle Lengh</string>
   </property>
  </widget>
  <widget class="QLabel" name="OtherValueL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Triangle Angle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TLenghLE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OVLE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>170</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SurfaceNameLE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface</string>
   </property>
  </widget>
  <widget class="QLabel" name="SurfaceNameL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface Name</string>
   </property>
  </widget>
  <widget class="QListView" name="PointGroupsLV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
